coriolis/tests/api/v1/data/replicas_get_merged_replica_values.yml

```diff
@@ -0,0 +1,78 @@
+
+- config:
+    replica:
+      origin_endpoint_id: "mock_origin_endpoint_id"
+      destination_endpoint_id: "mock_destination_endpoint_id"
+      source_environment: {'mock_source_key': 'mock_source_value'}
+      destination_environment:
+        storage_mappings: {'mock_destination_key': 'mock_destination_value'}
+      network_map: {'mock_network_key': 'mock_network_value'}
+      user_scripts: {'mock_scripts_key': 'mock_scripts_value'}
+      notes: "mock_notes"
+      origin_minion_pool_id: "mock_origin_minion_pool_id"
+      destination_minion_pool_id: "mock_destination_minion_pool_id"
+      instance_osmorphing_minion_pool_mappings:
+        mock_instance_1: "mock_pool_1"
+        mock_instance_2: "mock_pool_2"
+      user_scripts: "mock_user_scripts"
```

Comment:
Here you pass `user_scripts` twice (check line 10). I'm not sure whether you meant to put this in `updated_values`. If not, please remove it.